multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -102,9 +102,96 @@ def zero_shot_image_classification():
     }
 
 
+@automm_presets.register()
+def medium_quality_faster_train_object_detection():
```

Comment:
I think the preset name we agreed on is fast_inference and faster_inference.

Reply:
Just changed the naming. But in "faster" preset, I used validation loss as `val_metric`, which largely increased the training speed on small dataset (from 1474 seconds to 103 seconds on watercolor), and it didn't affect the inference speed. Comparing `faster` and `fast`, the inference fps difference is 85 vs 72, but the training time is 103 seconds v.s. 1645 seconds.

Reply:
I see. I think we can have additional presets if that preset is likely to be used. Because each problem type has its own characteristic. Detection might care about both training and inference time, but matching maybe care more about accuracy.